desktop/mac/release-notes/index.md

```diff
@@ -33,6 +33,40 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### Security
+
+- Update Docker Engine to v20.10.14 to address [CVE-2022-24769](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-24769)
```

Comment:
Should we add any of these CVEs to the Security announcements section in Docs - https://docs.docker.com/security/?

Reply:
Do we usually put Docker engine/upstream CVEs there @thaJeztah?